websites/Y/YouTube Music/metadata.json

```
@@ -20,7 +20,7 @@
 		"vi_VN": "Một dịch vụ phát nhạc với các album, đĩa đơn, video, bản remix, và các tiết mục trực tiếp chính thức và hơn nữa cho Android, iOS và máy tính. Tất cả đều tại đây."
 	},
 	"url": "music.youtube.com",
-	"version": "3.0.21",
+	"version": "3.1.0",
```

```suggestion
	"version": "3.0.22",
```
Changes are at patch-level